tools/code-owners-parser/CodeOwnersParser/CodeownersFile.cs

```diff
@@ -3,18 +3,48 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Text.Json;
 
 namespace Azure.Sdk.Tools.CodeOwnersParser
 {
     public static class CodeownersFile
     {
+
+        private static string teamUserBlobUri = $"https://{StorageConstants.AzureBlobAccountName}.blob.core.windows.net/{StorageConstants.AzureSdkWriteTeamsContainer}/{StorageConstants.AzureSdkWriteTeamsBlobName}";
```

Comment:
I'm OK with this be a default value but I think this needs to be an option that we can pass in at a command line parameter as well.

Reply:
As per our discussion, this is now an option that can be given to the parser and has been added as a command line option to the RetrieveCodeOwners executable.